src/amber/cli/templates/app/config/application.cr.ecr

```diff
@@ -7,4 +7,11 @@ Amber::Server.configure do |setting|
   setting.env = (ENV["AMBER_ENV"] ||= "development").to_s
   setting.log = ::Logger.new(STDOUT)
   setting.log.level = ::Logger::INFO
+
+  setting.session = {
+    :key       => "<%= @name %>.session",
+    :store     => :signed_cookie,
+    :expires   => 0,
+    :secret    => "<%= SecureRandom.hex(64) %>"
```

Comment:
This doesn't actually fix the issue because this secret doesn't actually get used by the current implementation. It gets passed into session::cookie_store but is abandoned there. New keys are then generated by signed cookie and encrypted_cookie which change every time it restarts. I still have a decent amount of work to make it use my new secret_key_base and I'd love to pair.